test/indexer/test_hca_indexer.py

```diff
@@ -239,13 +239,14 @@ def test_bundle_downgrade(self):
         self._index_canned_bundle(self.new_bundle)
         self._assert_new_bundle(num_expected_old_contributions=0)
         self._index_canned_bundle(self.old_bundle)
-        self._assert_old_bundle(num_expected_new_contributions=4, ignore_aggregates=True)
-        self._assert_new_bundle(num_expected_old_contributions=4)
+        self._assert_old_bundle(num_expected_new_contributions=5, ignore_aggregates=True)
+        self._assert_new_bundle(num_expected_old_contributions=5)
 
     def _assert_old_bundle(self, num_expected_new_contributions=0, ignore_aggregates=False, ignore_deletes=False):
         num_actual_new_contributions = 0
         hits = self._get_hits()
-        self.assertEqual(4 + 4 + num_expected_new_contributions, len(hits))
+        entity_contributions = 5
+        self.assertEqual(entity_contributions + entity_contributions + num_expected_new_contributions, len(hits))
```

Comment:
Contrary to what I said earlier this week, I think I like the previous version better.

```suggestion
        # Five entities (two files, one project, one sample and one bundle)
        # One contribution and one aggregate per entity
        self.assertEqual(5 + 5 + num_expected_new_contributions, len(hits))
```

The repeat the first comment wherever 5 occurs, and both comments wherever 5 + 5 occurs.